packages/vue-components/src/questions/Question.vue

```diff
@@ -23,7 +23,7 @@
               <span
                 v-for="keyword in keywordsSplitTrimmed()"
                 :key="keyword"
-                class="badge badge-pill badge-light fw-normal"
+                class="badge rounded-pill bg-light fw-normal"
```

Comment:
Will this need the `text-dark` as well?

```suggestion
                class="badge rounded-pill bg-light text-dark fw-normal"
```